style.css

```diff
@@ -458,3 +466,105 @@
 hr {
   width: 100%;
 }
+
+.contact-form {
+  margin-top: 100px;
+  border-top-left-radius: 68px;
+  display: flex;
+  flex-direction: column;
+  width: 373px;
+  height: 779px;
+  background: #6070ff;
+  background-repeat: no-repeat;
+}
```

Comment:
- Hey, this section is missing the background image and the black horizontal line as seen in the design from Figma. Please, refer to the Figma design to implement this missing part of the design to make your design close to the Figma design as much as you can.